db/system_keyspace.cc

```diff
@@ -259,6 +259,7 @@ schema_ptr system_keyspace::topology_requests() {
         return schema_builder(NAME, TOPOLOGY_REQUESTS, std::optional(id))
             .with_column("id", timeuuid_type, column_kind::partition_key)
             .with_column("initiating_host", uuid_type)
+            .with_column("request_type", utf8_type)
```

Comment:
IIRC:
- new columns can only be added at the end
- cluster feature is needed to ensure that new column is written to only after all nodes upgrade, unless something else is already ensuring that

cc @tgrabiec

Reply:
> IIRC:
> 
> * new columns can only be added at the end

RIL that columns are sorted lexicographically, so it doesn't matter where you add them.

> * cluster feature is needed to ensure that new column is written to only after all nodes upgrade, unless something else is already ensuring that

right, otherwise we're jeopardizing upgrade rollback